docs/flink-getting-started.md

```diff
@@ -634,6 +634,26 @@ FlinkSink.forRowData(input)
 env.execute("Test Iceberg DataStream");
 ```
 
+### Streaming upsert write
+
+Iceberg supports `UPSERT` based on the primary key when writing streaming data into v2 table format. To enable it, create an `UPSERT` table or configuration it when building a writer.
```

Comment:
it is probably confusing to call it `UPSERT table`. It seems to me that the essence of this part is that there are two ways to enable upsert, which is already covered the `Write options` section below.